lib/ex_unit/lib/ex_unit/formatter.ex

```diff
@@ -386,9 +386,6 @@ defmodule ExUnit.Formatter do
 
         {left, right}
 
-      nil when formatter == :expr ->
-        {if_value(left, inspect), if_value(right, inspect)}
```

Comment:
This seems to be dead code? Dialyzer suggests that `formatter` is always a function and I agree.

Reply:
Agreed.